### DISCUSSION TITLE
Deploy emailSendDefinition [4.0.0]

### DISCUSSION THREAD

**Discussion by JoeyNovais** [Accenture/sfmc-devtools#384] [Q&A] [answered]
Hi,

I'm trying to deploy some `emailSendDefinition` using the version 4.0.0 of mcdev. Looking at the documentation it is supported but when I try I always get the same warning/error.
Here I tried the normal way but got this:
`14:16:28 warn:  ☇ skipping emailSendDefinition: folderType userinitiatedsends not supported for deployment` 
Then I'm like ok, I should create a new folder to make this work, so I did change the path in the `"r__folder_Path"` attribute thinking that the tool will create the folder for me but got:
`14:16:59 error:  ☇ skipping emailSendDefinition DEV_EMEA_HE-45-JNO-Test / DEV_EMEA_HE-45-JNO-Test: Dependent folder with Path='HCP_Engagement_Test' was not found on your BU`
So I don't know how to resolve this because I don't know how and if we can create other folders for those kind of activities. I also try to retrieve the folders to check if I could create it from the dev tool but I didn't found the solution.

Any ideas ? :)

**Comment by JoernBerkefeld** [comment, marked as answer]
@DougMidgley - for what reason did we exclude that? Can't remember. Seems we need to improve that error message

**Reply by JoeyNovais** [reply]
Hey ! Any news on that one ?

**Reply by JoernBerkefeld** [reply]
uhm, @DougMidgley ?

**Reply by JoernBerkefeld** [reply]
@DougMidgley - could you please have a look?

**Reply by JoernBerkefeld** [reply]
hi @JoeyNovais 
bit late but - I just looked at your question again and I think I misunderstood your issue. WARNINGS are just that. There COULD be an issue, but there isn't one just yet. ERRORs signal something did indeed go wrong.

The warning here was just supposed to tell you that folders for the type you tried to deploy (emailsenddefinition) cannot be deployed.
I created a PR that changes the wording of that warning to be more explicit about the implications:
https://github.com/Accenture/sfmc-devtools/pull/587 

**It will be released with 4.2.0**
